content/en/docs/tasks/configure-pod-container/translate-compose-kubernetes.md

```diff
@@ -29,13 +29,13 @@ Kompose is released via GitHub on a three-week cycle, you can see all current re
 
 ```sh
 # Linux
-curl -L https://github.com/kubernetes/kompose/releases/download/v1.26.0/kompose-linux-amd64 -o kompose
+curl -L https://github.com/kubernetes/kompose/releases/download/v1.33.0/kompose-linux-amd64 -o kompose
```

Comment:
Just curious if there's a specific reason this change references Kompose v1.33 instead of the latest release [v1.34 (here)](https://github.com/kubernetes/kompose/releases/tag/v1.34.0)?

Reply:
my oversight.  I didn't see 1.34

Reply:
Updated to 1.34